### PR TITLE
Tailwind: `Slider` b2b styles

### PR DESCRIPTION
**Pull request by OsamaAbdellateef**

### CODE REVIEW

File: themes/theme-b2b/src/components/Slider.styles.ts [hunk 1]
```diff
@@ -3,17 +3,16 @@ import { cva } from 'class-variance-authority';
 
 export const Slider: ThemeComponent<'Slider'> = {
   track: cva([
-    'absolute top-4 h-2 w-full',
     'rounded-lg border-none border-transparent',
-    'bg-bg-surface-raised text-transparent',
+    'bg-[#e3e3e3] text-transparent',
```
Comment (OsamaAbdellateef) [inline themes/theme-b2b/src/components/Slider.styles.ts:7]: @sarahgm  
hint: this color is used only here

Comment (sarahgm) [reply themes/theme-b2b/src/components/Slider.styles.ts:7]: ok which token would you use than? Because there should be tokens used :)

Comment (sarahgm) [reply themes/theme-b2b/src/components/Slider.styles.ts:7]: thats why we used bg-surface-raised to have a token

**Comment by sarahgm** [comment]
@OsamaAbdellateef conflicts :)

**Review by sarahgm — APPROVED** [review]

**Pull request merged**